packages/amplify-category-api/package.json

```diff
@@ -59,7 +59,7 @@
     "graphql-http-transformer": "5.2.48-beta.0",
     "graphql-key-transformer": "3.2.48-beta.0",
     "graphql-predictions-transformer": "3.2.48-beta.0",
-    "graphql-relational-schema-transformer": "2.21.15-beta.0",
+    "graphql-relational-schema-transformer": "2.21.15",
```

Comment:
```suggestion
    "graphql-relational-schema-transformer": "2.21.15-beta.0",
```

Reply:
That was deliberate. If there's no major version change I prefer betas only when they get ahead.

Couple of examples.
- main is `2.21.16` and cdkv2 is `2.21.15-beta.0` - we should pick `2.21.16` to move forward, beta release will be `2.21.17-beta.0`
- main is `2.21.15` and cdkv2 is `2.21.15-beta.0` - we should pick `2.21.15` to move forward, beta release will be `2.21.16-beta.0` (like in case of this PR)
- main is `2.21.16` and cdkv2 is `2.21.17-beta.0` - we should pick `2.21.17-beta.0` to move forward, beta release will be `2.21.17-beta.1` (i.e. beta is already ahead so we should keep it so that lerna increments that last number)

Reply:
I agree in 1st and 3rd . In second case , since main is 2.21.15 , would it be preferred to choose 2.21.15-beta.0  as there is no change in module, this will simply beta  release 2.21.15-beta.1

Reply:
The problem is that when we decide to GA then lerna will calculate that GA version for `2.21.15-beta.1` is `2.21.15` and that will clash with already existing `2.21.15` in npm.
If we resolve to `2.21.15` and ship `2.21.16-beta.0` then GA is automatic to `2.21.16`.
Please keep in mind that there are dependency changes on branch, that's why it keeps releasing those packages.